apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -38,6 +38,7 @@ export class CustomRushAction extends BaseRushAction {
   private _changedProjectsOnly: CommandLineFlagParameter;
   private _fromFlag: CommandLineStringListParameter;
   private _toFlag: CommandLineStringListParameter;
+  private _toVersionPolicy: CommandLineStringListParameter;
```

Comment:
>_toVersionPolicy [](start = 10, length = 16)

We are also adding support for using "." .. Perhaps we should just re-use the "--to" flag for this behavior so it is more generic.

Reply:
I want the parameter to clearly mean it needs a version policy name as input.

We can rethink about the set of the input. But not in this pullrequest. 

---
In reply to: [184535690](https://github.com/Microsoft/web-build-tools/pull/625#discussion_r184535690) [](ancestors = 184535690)